### PR TITLE
feat: Support custom names for api keys

### LINKED ISSUE #334: [Feature request] Add labels to API keys.
Is it possible to add any kind of identification to API keys to help manage them ? I'm using multiple keys and keeping track of each keys is becoming a problem.

### PR DESCRIPTION
**Pull request by sergeylanzman**
Closes #334.

This PR introduces a name field for each API key to enhance the management of multiple API keys.

- When creating a new API key, users are now required to provide a name.

- The provided name is saved in the database and displayed in the UI for better identification and organization.

- Includes a database migration to support the new field.

### CODE REVIEW

File: web/src/api/apiKeys.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { createClient, handleError, handleResponse } from "@/api/api.utils";
 interface ApiKey {
   id: string,
   label: string,
+  name: string,
```
Comment (valentindeaconu) [inline web/src/api/apiKeys.ts:7]: The field label was added here for this exact use case, but I left it undone on the server side. It is implemented only on the client side. Can you, please, choose one of them (either remove the name and use the label, or remove the label and keep the name one)?

**Review by valentindeaconu — CHANGES_REQUESTED** [review]
Thank you for this! I was also working on something a bit more complex on a different branch, but I think your solution is cleaner and can be delivered faster, so I'm ok with merging it.

I think we will need to also update the E2E tests (apply the migration on existing databases and dump them again). Are you able to also fix them?

**Comment by sergeylanzman** [comment]
Thanks, i dont find how i can running tests on local machine, so starting debug with github PR

**Comment by sergeylanzman** [comment]
Done with unit tests and e2e

**Review by valentindeaconu — APPROVED** [review]
The changes look good to me. Thank you again for this contribution!

**Pull request merged**